test/runtests.jl

```diff
@@ -39,4 +39,38 @@ facts("Testing basic interface") do
     end
 end
 
+
+facts("Test utilities") do
+    context("test row_kron") do
+        h = ["a" "b"; "c" "d"]
+        z = ["1" "2" "3"; "4" "5" "6"]
+        want = ["a1" "a2" "a3" "b1" "b2" "b3"; "c4" "c5" "c6" "d4" "d5" "d6"]
+        @fact GMM.row_kron(h, z) => want
+
+        # now test on some bigger matrices
+        a = randn(400, 3)
+        b = randn(400, 5)
+        out = GMM.row_kron(a, b)
+        @fact size(out) => (400, 15)
+
+        rows_good = true
+        for row=1:400
+            rows_good = out[row, :] == kron(a[row, :], b[row, :])
+        end
+        @fact rows_good => true
```

Comment:
I didn't want to generate 400 tests for this simple test case. So what I mean to do was `rows_good &= out...` I'll fix that.